docs/usage.md

```diff
@@ -1,9 +1,9 @@
 | [Home](../README.md) |
-|--------------------------------------------|
+|----------------------|
 
 # Usage
 
-The Record Card widget helps visualize records based on a specified data source. An example of using the record widget would be displaying a list of tasks in the Record Card view on the Production and Development tab of the Continuous Delivery page.
+The **Record Card** widget helps visualize records based on a specified data source. An example of using the record widget would be displaying a list of tasks in the **Record Card** view on the Production and Development tab of the Continuous Delivery page.
```

Comment:
Can we add this as an example you can use it with continuous delivery.

It looks like this widget is only meant for continuous delivery.

Reply:
Done.